### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by lgarvey**
Initial project set-up

### CODE REVIEW

File: runtime.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+python-3.6.8
```
Comment (marcofucci) [inline runtime.txt]: python-3.7.2?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+#
+# Check https://circleci.com/docs/2.0/language-python/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      # use `-browsers` prefix for selenium tests, e.g. `3.6.1-browsers`
+      - image: circleci/python:3.6.1
```
Comment (marcofucci) [inline .circleci/config.yml]: python:3.7

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      - image: circleci/postgres:9.6
```
[the remark this answers is not in the document]
Comment (lgarvey) [reply .circleci/config.yml]: I've bumped this to 10.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+# A Dunn & Bradstreet caching microservice
+
+## Dependencies
+
+Python 3.7
+Django 2+
```
[the remark this answers is not in the document]
Comment (lgarvey) [reply README.md]: Added.

File: runtime.txt (hunk 1 above)
Comment (lgarvey) [reply runtime.txt]: Bumped.

File: .circleci/config.yml (hunk 2 above)
Comment (lgarvey) [reply .circleci/config.yml]: I've removed extraneous comments from the circleci file.

**Pull request merged**